Descriptions/Curation.md

```diff
@@ -0,0 +1,62 @@
+# Data Curation
+
+Here we declare the total data curation pipeline to provide consistent modeling development and prediction results. We use the same data curation rules for both training sets and prediction sets as they should be equally curated.
+For model development, we explore the experimental background from which the data was obtained, and for prediction, we strongly recommend using the applicability domain of developed models to ensure if they are applicable to studied biochemical problems.
+
+## Common pipeline for data from different sources
+
+The preliminary curation process guarantees that the data from all used sources is compatible with each other and standardized for further model development. We focus on the following steps:
+1. Removing data that is not associated with both structural data and property/activity value.
+2. Basically, structural information and a value or binary result are sufficient for modeling, but additional fields of data points are preferable to handle when present:
+- Source type - a patent, a manuscript, a commercial database with assays - these categories can provide valuable information as they usually study different chemical classes, and the quality and volume of data can vary significantly.
```

Comment:
Don't divide part of the sentence using '-'. Just:
```
Source type. 
A patent, manuscript, commercial database with assays. 
These sources offer valuable information, typically covering diverse chemical classes, with significant variations in data quality and volume.
```
So the whole structure will be just:
```
Field
What is included
Why we actually need it
```

Reply:
Same for all remaining points

Reply:
+